apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QMapTests.java

```diff
@@ -0,0 +1,31 @@
+package io.stargate.sgv2.it;
+
+public interface RestApiV2QMapTests {
```

Comment:
Not sure why there's API/impl if it covers both tests -- adding Javadoc would be good, explaining the reasoning (it's for different default setting?)

Reply:
Added java doc